nginx.tmpl

```diff
@@ -12,14 +12,24 @@ events {
 http {
   proxy_cache_path /tmp/nginx_cache levels=1:2 keys_zone=image_cache:10M max_size=100M inactive=40d;
 
+  # This prevents internal IP addresses from leaking when providing an empty host header
+  # requests where the Host header does not match *.ndla.no will be closed (nginx closes the connection when returning a 444)
+  server {
+    listen 79;
```

Comment:
Dette er kun for http? Hva skjer om man får en tom host-header når man kommer inn på https?